lib/src/crypto/src/crypto.dart

```diff
@@ -0,0 +1,48 @@
+import 'dart:typed_data';
+
+import '../../error/error.dart';
+import '../../generated/platform_constants.dart';
+import '../../platform/platform.dart';
+import '../crypto.dart';
+
+/// Utility methods for creating keys ([generateRandomKey]) and configuration
+/// objects ([CipherParams]) for symmetric encryption.
+class Crypto {
+  static const DEFAULT_ALGORITHM = 'aes';
+  static const DEFAULT_BLOCK_LENGTH_IN_BYTES = 16;
+  static const DEFAULT_KEY_LENGTH = 256;
+  static const DEFAULT_MODE = "cbc";
+  static const KEY_TYPE_ERROR_MESSAGE =
+      'A key must either be a String or a Uint8List.';
+
+  /// Gets the CipherParams which can be used to with [RestChannelOptions] or
+  /// [RealtimeChannelOptions] to specify encryption.
+  ///
+  ///  Pass a [String] containing a base64 encoded key or a [Uint8List]
+  ///  containing raw bytes for the key. If you have a password, do not use it
```

Comment:
I think it's worth stating explicitly that the key length here must match the default of 256 bits.

Reply:
I actually thought the user can use any size key, and I've tested this and the app crashes if they set a key size which isn't 256 bits. **Thanks**

a6ac0c33ffd6012db31939ba482d19be9f1d02d4